components/p3a/brave_p3a_uploader.cc

```diff
@@ -0,0 +1,87 @@
+/* Copyright 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/p3a/brave_p3a_uploader.h"
+
+#include "base/base64.h"
+#include "net/base/load_flags.h"
+#include "services/network/public/cpp/shared_url_loader_factory.h"
+#include "services/network/public/cpp/simple_url_loader.h"
+
+namespace brave {
+
+namespace {
+
+// TODO(iefremov): Provide more details for the traffic annotation.
+net::NetworkTrafficAnnotationTag GetNetworkTrafficAnnotation() {
+  return net::DefineNetworkTrafficAnnotation("metrics_report_uma", R"(
+      semantics {
+        sender: "Brave Privacy-Preserving Product Analytics Uploader"
+        description:
+          "Report of anonimized usage statistics. The desciprtion is WIP."
```

Comment:
@jsecretan @davidtemkin I would appreciate your help with better wording for the traffic annotation - this is a human readable description of the data we upload

Reply:
Updated 👍